package.json

```diff
@@ -28,7 +29,9 @@
     "mocha": "^5.0.5",
     "ts-node": "^5.0.1",
     "tslint": "^5.9.1",
-    "typescript": "^2.8.3"
+    "typescript": "^2.8.3",
+    "dotenv": "^5.0.1",
```

Comment:
nit: let npm take care of adding packages to your package/package-lock.json for you